### ISSUE TITLE
Telegraf Inputs Plugin helper function Can't render single brackets in toml

### ISSUE THREAD

**Issue by Mr-Howard-Roark** [influxdata/helm-charts#148]
The configmap template for input plugins calls this helper function: https://github.com/influxdata/helm-charts/blob/master/charts/telegraf/templates/configmap.yaml#L21

That helper function then has a convoluted way of converting yaml into toml: https://github.com/influxdata/helm-charts/blob/master/charts/telegraf/templates/_helpers.tpl#L181-L268

I have a use case for using the cisco_telemetry_mdt plugin: https://github.com/influxdata/telegraf/tree/master/plugins/inputs/cisco_telemetry_mdt

Everything works well except for aliases, which needs to be converted to:
```
 [inputs.cisco_telemetry_mdt.aliases]
   ifstats = "ietf-interfaces:interfaces-state/interface/statistics"
```

However, no matter what I try it ends up getting converted to the below with two brackets:
```
 [[inputs.cisco_telemetry_mdt.aliases]]
   ifstats = "ietf-interfaces:interfaces-state/interface/statistics"
```
That ultimately results in the telegraf pod crashing with the error "cannot unmarch TOML array table into map[string]string (need slice)

*Question:*
Does anyone have advice on how to have single brackets get rendered instead of double brackets?  Currently I am doing the below, which looks logical to me:
```
inputs:
  - cisco_telemetry_mdt:
      transport: "grpc"
      service_address" ":57000"
      aliases:
        ifstats: "ietf-interfaces:interfaces-state/interface/statistics"
```

**Comment by pravindamodaran** [comment]
@Mr-Howard-Roark I faced the same problem. There is a PR opened with alternate options to specify config file for telegraf
https://github.com/influxdata/helm-charts/pull/117

Since its not merged yet, you may have to have a local helm chart customized with these changes until its merged

**Comment by Mr-Howard-Roark** [comment]
Ok thanks, I will go that route then if there's no intention of updating the way the toml conversion works.  It does seem that using a raw telegraf config file is much simpler anyway since the toml conversion being done, while very cool, is hard to follow.

**Comment by alespour** [comment]
Fixed in #413 

Rendered output:
```
    [[inputs.cisco_telemetry_mdt]]
      service_address = ":57000"
      transport = "grpc"
      [inputs.cisco_telemetry_mdt.aliases]
        ifstats = "ietf-interfaces:interfaces-state/interface/statistics"
```
